scripts/apps/search/components/FetchedDesksInfo.tsx

```diff
@@ -71,22 +71,27 @@ export class FetchedDesksInfo extends React.Component<any, any> {
             const desk = this.state.desks[0];
             const name = this.formatDeskName(desk.desk.name);
 
-            items.push(React.createElement('dd', {key: 'dd1'}, desk.isUserDeskMember ?
-                React.createElement('a', {onClick: this.openDesk(desk)}, name) :
-                React.createElement('span', {className: 'container'}, name),
-            ));
+            items.push(
+                <dd key="dd1">
+                    {
+                        desk.isUserDeskMember ?
+                            <a onClick={this.openDesk(desk)}>{name}</a> :
```

Comment:
If there's no `href` it should be a button. Besides being semantically incorrect it also breaks keyboard navigation. Links without `href` attribute don't receive focus.